Storage/REST-API.md

```diff
@@ -0,0 +1,185 @@
+# RESTful binding for Storage, RFC
+
+This document describes the RESTful API with CRUD functionality for Current Storage.
+
+### Bindings semantics
+`TODO: Describe storage field <-> endpoint bindings.`
+
+### Response schema
+
+The response body is always either a valid JSON or empty with `204 (No content)` error code.
```

Comment:
Also, depending on how fancy you want the style of this spec to be, it may be worth explicitly mentioning the content-type of responses. But not very useful again, as the API seems to assume only custom clients which will expect only one type of representation anyway.